spec/osut_tests_spec.rb

```diff
@@ -309,119 +309,110 @@
 
   it "checks if a set holds a construction" do
     mdl = OpenStudio::Model::Model.new
-    v = mdl.getVersion.versionIdentifier.split(".").join.to_i
     translator = OpenStudio::OSVersion::VersionTranslator.new
 
-    if v < 350 # 5ZoneNoHVAC holds 1x OS:Material:AirWall, deprecated > 3.4.0
-      file = File.join(__dir__, "files/osms/in/5ZoneNoHVAC.osm")
-      path = OpenStudio::Path.new(file)
-      model = translator.loadModel(path)
-      expect(model.empty?).to be(false)
-      model = model.get
-
-      expect(mod1.clean!).to eq(DBG)
-
-      t1  = "roofceiling"
-      t2  = "wall"
-      cl1 = OpenStudio::Model::DefaultConstructionSet
-      cl2 = OpenStudio::Model::LayeredConstruction
-      n1  = "CBECS Before-1980 ClimateZone 8 (smoff) ConstSet"
-      n2  = "CBECS Before-1980 ExtRoof IEAD ClimateZone 8"
-      m1  = "'#{n2}' #{cl2}? expecting #{cl1} (OSut::holdsConstruction?)"
-      m5  = "Invalid 'surface type' arg #5 (OSut::holdsConstruction?)"
-      m6  = "Invalid 'set' arg #1 (OSut::holdsConstruction?)"
-
-      set = model.getDefaultConstructionSetByName(n1)
-      expect(set.empty?).to be(false)
-      set = set.get
-
-      c = model.getLayeredConstructionByName(n2)
-      expect(c.empty?).to be(false)
-      c = c.get
+    file = File.join(__dir__, "files/osms/in/5ZoneNoHVAC.osm")
```

Comment:
Removed the _OS:Material:AirWall_ from the _5ZoneNoHVAC.osm_ (kept tripping over this for SDK versions > 340).